app/services/git_hub_service.rb

```diff
@@ -9,6 +9,16 @@ def get_repos
     JSON.parse(response.body, symbolize_names: true)
   end
 
+  def retrieve_followers
+    response = conn.get('followers')
+    JSON.parse(response.body, symbolize_names: true)
+  end
+
+  def retrieve_following
+    response = conn.get('following')
+    JSON.parse(response.body, symbolize_names: true)
+  end
```

Comment:
The 3 above methods: get_repos, retrieve_followers, and retreive_following are all very similar.  To DRY this up, what do you think about refactoring to one method that took one argument, a string, of what you are retrieving? ("repos", "followers", "following")

Reply:
Addressed in latest commit